### PR TITLE
update of mica permanents URI

### PR DESCRIPTION
**Pull request by gloucklegnou**
addition of support for JSON-LD format for resources

### CODE REVIEW

File: mica/ontology/.htaccess [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Options +FollowSymLinks
+mv Options +FollowSymLinks
```
Comment (davidlehn) [inline mica/ontology/.htaccess]: typo?

Comment (gloucklegnou) [inline mica/ontology/.htaccess]: yes just remove this 'mv' typo thanks

**Comment by davidlehn** [comment]
The conflicts need to be resolved.  It's due to a commit I did the other day to normalize the line endings:
https://github.com/perma-id/w3id.org/commit/db8a33777acefde085ccf7a6a73f2de33269ed0f
If you could merge in upstream and resolve conflicts that that would fix this.  Also I'd suggest trying to just use LF endings vs having a mixed file.  If you have editorconfig support, there's now a top level file in the repo that will set this up for you.

**Comment by natetem** [comment]
We resolved the problems of line endings. 
We would like to merge our project in order to update the URIs. 

Check if every thing is OK?

**Comment by dgarijo** [comment]
There are still 2 conflicting files.
If the problem persists, I suggest updating the project to the latest revision and then push your changes on top

**Comment by natetem** [comment]
I can not solve the conflict problem on GitHub. I suggest you to update the latest version and I will ask a new pull request. 

Thank you.

**Comment by dgarijo** [comment]
@natetem,
I don't understand your answer. I already am with the latest version of the branch. You are the one who has submitted a pull request with conflicts. Therefore, I suggest you to update your repository and try again.
Thanks

**Comment by dgarijo** [comment]
Still cannot merge the branch due to conflicts. Please resolve them before I can merge the branch.

**Comment by natetem** [comment]
@dgarijo 
I resolve the conflicts, I think every thing is ok. 
You can accept our pull request. 

Thanks!
